scenes/gameplay/pause-layer/pause-layer.gd

```diff
@@ -32,3 +37,9 @@ func _on_Main_Menu_pressed():
 	Game.change_scene("res://scenes/menu/menu.tscn", {
 		'show_progress_bar': false
 	})
+
+
+func _on_PauseButton_pressed():
+	resume_option.grab_focus()
+	get_tree().paused = true
+	pause.show()
```

Comment:
Can we simply call `pause_game()` and avoid duplication? What do you think?

Reply:
Yes, it's my fault, I didn't see the `pause_game()` function.